VAMobile/src/components/FormWrapper/FormWrapper.test.tsx

```diff
@@ -185,70 +154,7 @@ context('FormWrapper', () => {
           expect(onSaveSpy).toHaveBeenCalled()
         })
       })
```

Comment:
I believe there is another ticket in the backlog #5859 to update these unit tests below to be able to function. So please take a look and see if there are any other tests that you can possibly test again or things that may be able to be tested. Once you have done so feel free to close out 5859 as this ticket should cover that update

Reply:
won't fix.